### DISCUSSION TITLE
Moving Tool missing in the Tool Toolbox?

### DISCUSSION THREAD

**Discussion by Displayel** [audacity/audacity#2232] [Q&A] [answered]
Hello, 
can anybody help on this?  The Moving Tool or "Verschiebewerkzeug" in german has disappeared after my last update, where is it now? 
<img width="109" alt="Yes" src="https://user-images.githubusercontent.com/10302495/144724796-d373c48c-b7f4-4553-a63f-9e55c098adb1.png">
 
It looks like this now
<img width="100" alt="No" src="https://user-images.githubusercontent.com/10302495/144724813-1df3fbe6-236a-4d18-94d0-005e04947c8b.png">

I´m on 3.1.2.

**Comment by LWinterberg** [comment, marked as answer]
It has been removed. You can move clips around without this tool now by just dragging the top of  the clips.
